bool dirL
bool dirR
int32 speedL
int32 speedR
int32 n_cntL
int32 n_cntR
int32 battery_12
int32 battery_24
bool safety_button
bool field1
bool field2
bool field3
bool field4
